Assets/Scripts/UI/PauseScrn/HighlightEnlarge.cs

```diff
@@ -0,0 +1,69 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+public class HighlightEnlarge : MonoBehaviour {
+	public GameObject bigNote0;
+	public GameObject bigNote1;
+	public GameObject bigNote2;
+	public GameObject bigNote3;
+	public GameObject bigNote4;
+	public GameObject SelectedNote_Panel;
+	public Button back;
+
+	// Use this for initialization
+	void Start () {
+		SelectedNote_Panel.gameObject.SetActive (false);
+		deactivateAll ();
+
+
+	}
+
+	// Update is called once per frame
+	void Update () {
+		
+	}
+
+	void deactivateAll(){
+		bigNote0.SetActive (false);
+		bigNote1.SetActive (false);
+		bigNote2.SetActive (false);
+		bigNote3.SetActive (false);
+		bigNote4.SetActive (false);
+	}
+	public void Return(){
+
+			SelectedNote_Panel.SetActive (false);
+			deactivateAll ();
+			back.interactable = true;
+			back.Select ();
+		//SelectedNote_Panel.gameObject.SetActive (false);
+		//deactivateAll ();
+	}
+
+	public void NotezeroSelected(){
```

Comment:
We should really put this in an array... this is bad implementation. But time rush so we can fix later.